src/components/scenes/CoinRankingDetailsScene.tsx

```diff
@@ -218,6 +271,176 @@ const CoinRankingDetailsSceneComponent = (props: Props) => {
     return rows
   }
 
+  const getWallet = async () => {
+    if (edgeAsset == null) return
+    const { pluginId, tokenId } = edgeAsset
```

Comment:
Do we need to enable the token? It seems like we don't handle the case where the wallet exists for the chain, but the token itself isn't active.